### ISSUE TITLE
[V5] Crash on iOS after Reload with enabled Debugging

### ISSUE THREAD

**Issue by sintylapse** [realm/realm-js#2836]
## Issue
After upgrading to v5 after enabling debugging, app crashes with error `EXC_BAD_ACCESS`, but everything okay after reload, then it will crash again after next reload.

<img width="944" alt="Screen Shot 2020-04-23 at 16 09 05" src="https://user-images.githubusercontent.com/12295016/80119400-566e2580-8592-11ea-9bb8-c4f512a16f79.png">

## Expected Results
Should not crash after each second reload when debugging is enabled

## Actual Results
Crash occurs each second reload when debugging is enabled

## Steps to Reproduce
On iOS simulator enable Debug from dev menu

## Version of Realm and Tooling
- Realm JS SDK Version: 5.0.3
- React Native: 61.5
- Which debugger for React Native: React Native Debugger

**Comment by noldidrita** [comment]
I'm having this issue as well for quite some time. I've downgraded to 4.0.0 just because of this, as it made it very hard to work with.

**Comment by blagoev** [comment]
Hi, yes we have this issue https://github.com/realm/realm-js/issues/2799
In the mean time you can use Safari to debug you RN realm application

**Comment by sintylapse** [comment]
Closing as duplicate

**Comment by sintylapse** [comment]
The issue was resolved after updating GCDWebServer to 3.5.4 in Podfile.lock
```
  - GCDWebServer (3.5.4):
    - GCDWebServer/Core (= 3.5.4)
  - GCDWebServer/Core (3.5.4)
```
